docs/src/Button.doc.js

```diff
@@ -281,38 +284,52 @@ card(
     <Combination
       id="color"
       name="Color"
-      color={[
-        'gray',
-        'red',
-        'blue',
-        'white',
-        'transparent',
-        'transparentWhiteText',
-      ]}
+      color={['gray', 'red', 'blue', 'white', 'transparent']}
     >
-      {(props, i) => {
-        let color;
-        if (props.color === 'transparentWhiteText') {
-          color = 'darkWash';
-        }
-        if (props.color === 'transparent') {
-          color = 'lightWash';
-        }
-        return (
-          <Box color={color}>
-            <Button
-              id={`example-${i}`}
-              onChange={() => {}}
-              {...props}
-              text="Button"
-            />
-          </Box>
-        );
-      }}
+      {(props, i) => (
+        <Button
+          id={`example-${i}`}
+          onChange={() => {}}
+          {...props}
+          text="Button"
+        />
+      )}
     </Combination>
   </>
 );
 
+card(
+  <Example
+    name="Color: transparent"
+    id="transparent"
+    defaultCode={`function Example() {
+  const [selected, setSelected] = React.useState(false);
+  return (
+    <Stack gap={1}>
+      <Button text="Toggle Text Color" onClick={() => setSelected(!selected)} inline selected={selected}/>
+      <Box column={3}>
```

Comment:
@AlbertCarreras this part of the docs don't show up for me. Probably because there is no explicit width on the image container.

![image](https://user-images.githubusercontent.com/127199/92664453-e3531a80-f2b8-11ea-8009-b97ab1080475.png)

This does fix it for me:
```
<Box width={300}>
        <Image
          alt="Tropic greens: The taste of Petrol and Porcelain | Interior design, Vintage Sets and Unique Pieces agave"

```

Reply:
Damn, I fixed that but it looks like I didn't commit correctly. Apologies.

Reply:
DONE